commands/ping.go

```diff
@@ -17,41 +19,65 @@ var (
 		Use:   "ping",
 		Short: "Check API connection",
 		Long:  `Tests the connection to the API`,
-		Run:   ping,
+		Run:   runPingCommand,
 	}
 )
 
 func init() {
 	RootCommand.AddCommand(PingCommand)
 }
 
-// ping checks the API connections
-func ping(cmd *cobra.Command, args []string) {
-	u, err := url.Parse(cmdAPIEndpoint)
+// runPingCommand executes the ping() function
+// and prints output in a user-friendly way
+func runPingCommand(cmd *cobra.Command, args []string) {
+	duration, err := ping(cmdAPIEndpoint)
 	if err != nil {
-		fmt.Println(err.Error())
+		fmt.Println(color.RedString("API connection is bad"))
```

Comment:
Currently the user gets the errors details as an additional line:

```
API connection is bad
Get http://nonexisting/v1/ping: dial tcp: lookup nonexisting on 8.8.8.8:53: no such host
```

I'll happily change the wording to "Could not reach API".